1.0/php/sample_app/composer.json

```diff
@@ -3,6 +3,7 @@
   "description": "gRPC example for PHP",
   "require": {
     "php": ">=5.5.0",
-    "grpc/grpc": "v1.0.0"
+    "grpc/grpc": "v1.2.0",
+    "stanley-cheung/protobuf-php": "v0.6"
```

Comment:
This `stanley-cheung/protobuf-php` dependency is no longer needed after grpc version 1.1.0+